cluster-configuration/services-configuration.yaml

```diff
@@ -117,3 +117,20 @@ prometheus:
 pylon:
   # port of pylon
   port: 80
+
+
+role:
```

Comment:
Can Separate this field into corresponding service charts?

Reply:
Resolved